src/main/java/com/browserstack/automate/ci/common/BrowserStackBuildWrapperOperations.java

```diff
@@ -129,6 +130,15 @@ public void buildEnvVars(Map<String, String> env) {
             }
         }
 
+        String grr_region = env.get(BrowserStackEnvVars.GRR_JENKINS_KEY);
+        if ( grr_region != null && Constants.GRR_AUTO_REGION_VS_APIURL.containsKey(grr_region.toLowerCase()) ) {
+            logEnvVar(BrowserStackEnvVars.GRR_JENKINS_KEY, grr_region);
+            System.setProperty(BrowserStackEnvVars.AUTOMATE_API_ENV_KEY, Constants.GRR_AUTO_REGION_VS_APIURL.get(grr_region.toLowerCase()));
```

Comment:
redundant code in line 136, 138. same with app-automate block.

Reply:
App Automate has separate system property. So, has to do it for it separately only. Have moved it to a method instead.